red-hat-storage/ocs-ci
# Verify 'noobaa-service-monitor' does not exist after OCS upgrade

BZ link: https://bugzilla.redhat.com/show_bug.cgi?id=1973179  
Test Procedure:
  1.Upgrade OCS version [4.6->4.7]
  2.Check servicemonitors
  3.Verify 'noobaa-service-monitor' does not exist
Signed-off-by: Oded Viner <oviner@redhat.com>
